semantic-conventions/CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+# Semantic Conventions build tools
+
+This document provides `semantic-convention` specific information for contributions.
+
+Read OpenTelemetry project [contributing
+guide](https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md)
+for general information about the project.
+
+## Development setup
+
+### Prerequisites
+
+- Python
+- Docker (optional)
+
+### Building
+
+Any changes to the build process should be reflected in this repositiory's `.github/workflows/semconvgen.yml` workflow, in addition to this document and the [CHANGELOG](CHANGELOG.md).
+
+For local development, it may be helpful to use an isolated environment such as [venv](https://docs.python.org/3/library/venv.html), or whatever your preferred IDE may offer. Doing so will avoid polluting your system's path with dependencies specific to this project, and allow for more reproducible builds. For detailed instructions setting up python, see [venv setup](#venv-setup)
```

Comment:
```suggestion
For local development, use an isolated environment ([venv](https://docs.python.org/3/library/venv.html)). Doing so will avoid polluting your system's path with dependencies specific to this project, and allow for more reproducible builds. For detailed instructions setting up python, see [venv setup](#venv-setup)
```

That's really the only mechanism that Python supports, IDEs can just offer nicer UIs to manage them.